cmd/pmm-dump/main.go

```diff
@@ -97,6 +98,11 @@ func main() { //nolint:gocyclo,maintidx
 				Default(fmt.Sprintf("%v=90,%v=90,%v=30", transferer.ThresholdCPU, transferer.ThresholdRAM, transferer.ThresholdMYRAM)).String()
 
 		stdout = exportCmd.Flag("stdout", "Redirect output to STDOUT").Bool()
+		// encryption related
+		noEncryption = cli.Flag("no-encryption", "Disable encryption").Default("false").Bool()
```

Comment:
It results in the following --help output:

```
      --[no-]no-encryption     Disable encryption ($PMM_DUMP_NO_ENCRYPTION)
```
This is confusing. Should be --[no-]encryption with encryption enabled by default.